src/content/docs/alerts-applied-intelligence/applied-intelligence/anomaly-detection/automatic-anomalies.mdx

```diff
@@ -0,0 +1,26 @@
+---
+title: Automatic anomaly detection 
+tags:
+  - Alerts and applied intelligence
+  - Applied intelligence
+  - Proactive detection
+metaDescription: Learn how automatic automatic anomaly detection in New Relic notifies you of unusual app behavior. 
+---
+
+Automatic anomaly detection is the easiest way for you and your team to detect and analyze unusual behavior in your system. Automatic anomaly detection uses the following methods to detect anomalies in your app data:
```

Comment:
The second sentence I think can be tweaked a bit. Something like "It does this using the following methods:"

Reply:
done